threads.go

```diff
@@ -675,21 +707,30 @@ func (t *threads) startPulling() {
 	tick := time.NewTicker(PullInterval)
 	defer tick.Stop()
```

Comment:
Opt: maybe moving this after the initial pull, to avoid overlap.

Reply:
👍